packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/add/edit/logs-inputs-add-edit.controller.js

```diff
@@ -39,9 +41,25 @@ class LogsInputsAddEditCtrl {
         return enginesList;
       }, []);
     });
+    this.accountDetails.load().then(() => {
+      this.ovhSubsidiary = this.accountDetails.data.me.ovhSubsidiary;
+      this.$q.all([this.mainOffer.load(), this.catalog.load()]).then(() => {
```

Comment:
```suggestion
      return this.$q.all([this.mainOffer.load(), this.catalog.load()]);
      })
      .then(() => {
```

this way we can chain the promises !

Reply:
Actually, this looks like duplication, could you please refactor to DRY ?

Reply:
Suggested changed applied on three cases